src/styles/bloks/_card.scss

```diff
@@ -0,0 +1,107 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: ($card-with + 48px) * 4;
+  box-sizing: content-box;
+
+  &__card {
+    width: $card-with;
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $main-color;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    display: flex;
+    width: 100%;
+    height: 134px;
+    text-align: center;
+    align-self: center;
+    padding: 0 3px;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)